dataprofiler/tests/profilers/test_utils.py

```diff
@@ -124,3 +124,21 @@ def test_find_diff(self):
             "g": [None, 15]
         }
         self.assertDictEqual(expected_diff, utils.find_diff_of_dicts(dict1, dict2))
+
+    def test_get_capacity(self):
+        """
+        Checks to see if the get capacity function is operating appropriately.
+        """
+        # wrong unit input
+        with self.assertRaisesRegex(ValueError,
+                "Currently only supports the capacity unit in \[B, K, M, G\]"):
+            utils.get_capacity([], unit="wrong_unit")
+
+        # test with different data sizes
+        self.assertEqual(0, utils.get_capacity([]))
+        self.assertAlmostEqual(3.14712e-05,
+            utils.get_capacity(["This is test, a Test sentence.!!!"]))
```

Comment:
do these need to be almost equals? Don't we know what the exact value should be?
e.g. 33/1024**2

Reply:
changed